index.js

```diff
@@ -0,0 +1,29 @@
+document.addEventListener("DOMContentLoaded", () => {
+    fetch("https://cors-anywhere.herokuapp.com/https://official-joke-api.appspot.com/random_ten").then(res => {
+        if (!res.ok) {
+            throw Error(res.statusText + " was the error")
+        }
+        return res.json()
+    }).then(res => {
+        let ul = document.createElement("ul")
+        res.forEach((joke) => {
+            let li = document.createElement("li")
+            let p = document.createElement("p")
+            li.addEventListener("click", () => {
+                p.innerText = joke.punchline
+                li.appendChild(p)
+                li.addEventListener("dblclick", () => {
```

Comment:
The assignment asked for the second click to hide the punchline, not the double click. However it's cool you've discovered this event type!